src/BootstrapTable.js

```diff
@@ -344,6 +345,9 @@ class BootstrapTable extends Component {
     this._adjustTable();
     window.addEventListener('resize', this._adjustTable);
     this.refs.body.refs.container.addEventListener('scroll', this._scrollHeader);
+    if (this.props.showFooter) {
```

Comment:
@AllenFang This check, adds event listener only if `showFooter` is `true`. This avoids reference errors when footer is not required. Same has been done on line 360 :point_down:.